binary_search_tree/binary_search_tree.py

```diff
@@ -48,4 +48,9 @@ def contains(self, target):
 
 
```

Comment:
Great job on completing BST! I like the way you implemented your get_max function